docs/sources/next/using-k6-browser/migrating-to-k6-v0-52.md

```diff
@@ -0,0 +1,309 @@
+| Page           | setContent              |
+| Page           | setExtraHTTPHeaders     |
+| Page           | setInputFiles           |
+| Page           | setViewportSize         |
+| Page           | tap                     |
+| Page           | textContent             |
+| Page           | throttleCPU             |
+| Page           | throttleNetwork         |
+| Page           | title                   |
+| Page           | type                    |
+| Page           | uncheck                 |
+| Page           | viewportSize            |
+| Page           | waitForLoadState        |
+| Page           | waitForSelector         |
+| Page           | waitForTimeout          |
+| Request        | allHeaders              |
+| Request        | headerValue             |
+| Request        | headersArray            |
+| Request        | postData                |
+| Request        | resourceType            |
+| Request        | response                |
+| Request        | size                    |
+| Request        | timing                  |
+| Response       | allHeaders              |
+| Response       | body                    |
+| Response       | headerValue             |
+| Response       | headerValues            |
+| Response       | headersArray            |
+| Response       | json                    |
+| Response       | securityDetails         |
+| Response       | serverAddr              |
+| Response       | size                    |
+| Touchscreen    | tap                     |
+
+{{< /collapse >}}
+
+## Working with k6 check
+
+The k6 `check` API will not `await` promises, so calling a function that returns a `Promise`, for example `locator.textContent()`, inside one of the predicates will not work. Instead you will have to `await` and store the result in a variable _outside_ the `check`:
+
+For example, before:
+
+{{< code >}}
+
+<!-- eslint-skip -->
+
+```javascript
+check(page, {
+  header: (p) => p.locator('h2').textContent() == 'Welcome, admin!',
+});
+```
+
+{{< /code >}}
+
+And now:
+
+{{< code >}}
+
+<!-- eslint-skip -->
+
+```javascript
+const headerText = await page.locator('h2').textContent();
+check(headerText, {
+  header: headerText === 'Welcome, admin!',
+});
+```
+
+{{< /code >}}
+
+## Groups
+
+A note on [groups](https://grafana.com/docs/k6/<K6_VERSION>/using-k6/tags-and-groups/#groups), they don't work with async APIs either, there is no workaround as of yet. Here's the [GitHub issue](https://github.com/grafana/k6/issues/2728) that you can follow to keep up-to-date with relevant news on a group API that works with async APIs.
+
+# Where to get extra help
+
+With the release of k6 `v0.52`, you'll also be able to find the latest type definitions and updates to the k6 browser documentation. Refer to [Configure k6 IntelliSense](http://grafana.com/docs/k6/<K6_VERSION>/set-up/configure-k6-intellisense/) for details on how to work with the type definition file in your project.
```

Comment:
Adding a link here to the k6 browser documentation would be more valuable for users rather than them going and trying to find it.

Reply:
Added it in 9ac33402ab38c9cf4d2314571723cecd695c9e7f